eng/common/cross/toolchain.cmake

```diff
@@ -25,8 +25,6 @@ if(TARGET_ARCH_NAME STREQUAL "arm")
   set(CMAKE_SYSTEM_PROCESSOR armv7l)
   if(EXISTS ${CROSS_ROOTFS}/usr/lib/gcc/armv7-alpine-linux-musleabihf)
     set(TOOLCHAIN "armv7-alpine-linux-musleabihf")
-  elseif(EXISTS ${CROSS_ROOTFS}/usr/lib/gcc/armv6-alpine-linux-musleabihf)
-    set(TOOLCHAIN "armv6-alpine-linux-musleabihf")
```

Comment:
Why is this removed?

Reply:
My mistake, this wasn't supposed to be removed.